omero/sysadmins/publication.rst

```diff
@@ -0,0 +1,104 @@
+Complete workflow for hosting data for a publication
+====================================================
+
+The following describes the steps of a complete workflow for using OMERO to
+host public data associated with a publication. It is illustrated using an
+example publication from our lab in Dundee,
+`Schleicher et al, 2017 <http://dx.doi.org/10.1098/rsob.170099>`_.
+
+Group setup
+-----------
+
+#. Create a dedicated read-only group to host the raw data underlying the
+   publication (see :doc:`cli/usergroup`)
+#. Add all the authors of the paper to this new group.
+#. Once you have configured OMERO.web to create a public user (see below), add
+   the public user as a member of the newly created read-only group.
+
+Data setup
+----------
+
+#. Move the original images into the dedicated group using OMERO.web or
+   :doc:`OMERO.cli </users/cli/chgrp>`. The CLI is best used where Images or
+   Datasets are cross-linked to other Datasets or Projects in the original
+   group. The command ``bin/omero chgrp Project:$ID --include Dataset,Image``
+   cuts the cross-links in the original group and preserves the
+   Project/Dataset/Image hierarchy prepared for the move by the author.
+#. If you have used OMERO.figure to create your figures for publication, you
+   can always find the original data by using the 'info' tab, as shown in the
+   :help:`OMERO.figure Help guide <figure.html#info>` (OMERO.figure supports a
+   complete figure creation workflow, including exporting figures into image
+   processing applications for final adjustments - see the
+   :help:`OMERO.figure Help guide <figure.html>` for full details).
```

Comment:
If figures have been created in a non-public group and you want to make them public (as we did for Kat's paper) there are more steps to document, to move the figures to public group. It's not the nicest workflow via web UI but maybe not so bad via CLI?

Reply:
On CLI, you have to do either some hql queries or get the Figure IDs from the webclient by opening the + icon in attachments and finding the name of the Figure. Both are pretty "bad" too.

Reply:
To get the File-Annotation IDs you only have to open a figure in the browser and look at the URL. Then use the CLI to do the chgrp?

Reply:
The workflow for finding the figures from the web client isn't very user friendly either though - Petr just had to come and show me how to do it. So maybe not something to promote just yet

Reply:
@will-moore 
>To get the File-Annotation IDs you only have to open a figure in the browser and look at the URL. Then use the CLI to do the chgrp?

Yes, this is probably how I did it after your advice. But admittedly, this is not a nice workflow, reading IDs from the url. Also, you have to go one-by-one, with 3-4 clicks to get there. Does not support the idea of well-established workflow.

Reply:
Created https://trello.com/c/RwqBV6hN/81-moving-omerofigures-between-groups